queue_services/entity-filer/src/entity_filer/filing_processors/consent_continuation_out.py

```diff
@@ -0,0 +1,52 @@
+# Copyright © 2023 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""File processing rules and actions for the consent continuation out filing."""
+from contextlib import suppress
+from datetime import timedelta
+from typing import Dict
+
+import dpath
+import pytz
+from dateutil.relativedelta import relativedelta
+from legal_api.models import Business, Filing
+from legal_api.utils.datetime import datetime
+from legal_api.utils.legislation_datetime import LegislationDatetime
+
+from entity_filer.filing_meta import FilingMeta
+from entity_filer.filing_processors.filing_components import filings
+
+
+def process(business: Business, cco_filing: Filing, filing: Dict, filing_meta: FilingMeta):
+    """Render the consent continuation out filing into the business model objects."""
+    # update consent continuation out, if any is present
+    with suppress(IndexError, KeyError, TypeError):
+        consent_continuation_out_json = dpath.util.get(filing, '/consentContinuationOut/courtOrder')
+        filings.update_filing_court_order(cco_filing, consent_continuation_out_json)
+
+    cco_filing.order_details = filing['consentContinuationOut'].get('details')
+    expiry_date = get_expiry_date()
+    business.cco_expiry_date = expiry_date
+
+    filing_meta.consentContinuationOut = {}
+    filing_meta.consentContinuationOut = {**filing_meta.consentContinuationOut,
+                                          **{'expiry': expiry_date.isoformat()}}
+
+def get_expiry_date():
+    legislation_date_now = LegislationDatetime.now().date()
+    expiry_date = datetime.combine(legislation_date_now,
+                                   datetime.min.time(),
+                                   tzinfo=pytz.timezone("UTC")) \
+                           + relativedelta(months=6) \
+                           + timedelta(hours=8)
```

Comment:
This timedelta is the difference betweem PT and UTC, right? PT is UTC-8 for Standard Time, and UTC-7 for Daylight Savings time. (Have I mentioned how much I dislike coding around dates/times?)

Reply:
Yes, that's right.

Reply:
if we use the `eastern` timezone instead of `UTC`, the timedelta would be 3 hrs for both case.
How do you think about that?

Reply:
> if we use the `eastern` timezone instead of `UTC`, the timedelta would be 3 hrs for both case. What do you think about that?

we are already using UTC.  even if we did that we would need to update all the existing data in the database.

Reply:
> Doesn't "now" take into account Daylight/Standard times? I don't recall ever having to do this much work to code around it, but it's been a while since I've done it.

I think the main thing here is that we are trying to store midnight in UTC in a manner where we can also retrieve it for display purposes in pacific time.  We started just setting the timedelta to be 8 awhile back.  I'm assuming the reason for that is to just not deal with the PST/PDT calculations for anything we are representing as midnight and just default to PST.